www/caddy/pkg-descr

```diff
@@ -13,6 +13,15 @@ DOC: https://docs.opnsense.org/manual/how-tos/caddy.html
 Plugin Changelog
 ================
 
+1.7.4
+
+* Add: Layer4 OpenVPN matcher with mode, digest and static key support
+* Add: Layer4 Winbox matcher
+* Build: Update dependency to lang/go123
```

Comment:
Does caddy require this?

Reply:
The requirement is at least 1.22 for the layer4 module now. But from discussions I had using go 1.23 should be preferred, it is not a requirement though.

All my personal tests have been conducted with a go 1.23 build lately.

https://github.com/mholt/caddy-l4/commit/31ae08c39c2cec1f068901e4290436d4d40aad26